templates/index.html

```diff
@@ -1,61 +1,60 @@
-  	<meta property="og:site_name" content="{{ config.title }}">
-  	<meta property="og:description" content="{{ config.description }}">
-  	<meta property="og:title" content="{{ config.title }}">
-  	<meta property="og:image" content="{{ get_url(path='images/top-banner.png') }}">
-  	<!-- /Facebook -->
+		{% set social_title = config.title %}
+		{% if page %}
+			{% set social_title =  page.title ~ " | " ~ config.title %}
+		{% elif section and section.title %}
+			{% set social_title =  section.title ~ " | " ~ config.title %}
+		{% endif %}
 
-  	<!-- Twitter -->
-  	<meta name="twitter:card" content="summary_large_image">
-  	<meta name="twitter:site" content="{{ config.extra.twitter }}">
-  	<meta name="twitter:title" content="{{ config.title }}">
-  	<meta name="twitter:description" content="{{ config.description }}">
-  	<meta name="twitter:image" content="{{ get_url(path='images/top-banner.png') }}">
-  	<!-- /Twitter -->
+		<!-- Facebook -->
+		<meta property="og:url" content="{{ config.base_url | safe }}">
+		<meta property="og:type" content="website">
+		<meta property="og:site_name" content="{{ config.title }}">
+		<meta property="og:description" content="{{ config.description }}">
+		<meta property="og:image" content="{{ get_url(path='images/top-banner.png') | safe }}">
+		<meta property="og:title" content="{{ social_title }}">
+		<!-- /Facebook -->
 
-    <link rel="alternate" type="application/rss+xml" title="Software Sessions" href="https://feeds.transistor.fm/software-sessions" />
-
-	  {% endblock meta %}
-
-	  <style>html { visibility: hidden; opacity: 0; }</style>
-		<link rel="stylesheet" href="{{ get_url(path='style.css') }}">
-		<title>{% block title %}{{ config.title }}{% endblock title %}</title>
+		<!-- Twitter -->
+		<meta name="twitter:card" content="summary_large_image">
+		<meta name="twitter:site" content="{{ config.extra.twitter }}">
+		<meta name="twitter:description" content="{{ config.description }}">
+		<meta name="twitter:image" content="{{ get_url(path='images/top-banner.png') | safe }}">
+		<meta name="twitter:title" content="{{ social_title }}">
+		<!-- /Twitter -->
 	</head>
 	<body>
 		<header class="header">
 			<div class="header-content">
-				<h1 class="site-title"><a href="{{config.base_url}}">
+				<h1 class="site-title"><a href="{{config.base_url | safe}}">
 				software sessions
 				</a></h1>
 
 				<nav class="main-nav">
 					<ul>
 						<li><a href="{{config.base_url}}">Home</a></li>
 						<span> | </span>
-						<li><a href="{{config.base_url}}/episodes">Episodes</a></li>
+						<li><a href="{{ get_url(path="@/episodes/_index.md") }}">Episodes</a></li>
 						<span> | </span>
-						<li><a href="{{config.base_url}}/subscribe">Subscribe</a></li>
+						<li><a href="{{ get_url(path="@/subscribe/index.md") }}">Subscribe</a></li>
 						<span> | </span>
-						<li><a href="{{config.base_url}}/about">About</a></li>
+						<li><a href="{{ get_url(path="@/about/index.md") }}">About</a></li>
```

Comment:
it isn't super necessary to do links like that in templates but I find it helpful when some pages define their own URL path that are different from the file paths and I want to know where this content is

Reply:
Yeah, this does look better. So you would only use `config.base_url` for linking to the root path?